contracts/misc/WrappedTokenGatewayV3.sol

```diff
@@ -136,26 +140,26 @@ contract WETHGateway is IWETHGateway, Ownable {
    * @param permitS S parameter of ERC712 permit sig
    */
   function withdrawETHWithPermit(
-    address pool,
+    address,
     uint256 amount,
     address to,
     uint256 deadline,
     uint8 permitV,
     bytes32 permitR,
     bytes32 permitS
   ) external override {
-    IAToken aWETH = IAToken(IPool(pool).getReserveData(address(WETH)).aTokenAddress);
+    IAToken aWETH = IAToken(POOL.getReserveData(address(WETH)).aTokenAddress);
     uint256 userBalance = aWETH.balanceOf(msg.sender);
     uint256 amountToWithdraw = amount;
 
     // if amount is equal to uint(-1), the user wants to redeem everything
```

Comment:
```suggestion
    // if amount is equal to type(uint256).max, the user wants to redeem everything
```
*Very minor* update, both are valid but since we don't use uint(-1), thinking we should just update this.